src/sentry/api/serializers/models/release.py

```diff
@@ -158,6 +158,10 @@ def get_attrs(self, item_list, user, *args, **kwargs):
 
         release_metadata_attrs = self._get_commit_metadata(item_list, user)
 
+        release_projects = defaultdict(list)
+        project_releases = ReleaseProject.objects.filter(release__in=item_list).select_related('project')
```

Comment:
@mattrobenolt – This serializer is now doing a lot of extra work by default (related authors, projects, commit counts, etc). Should we maybe amend the API to like, optionally bring in this data (e.g. `&includes=authors&includes=projects`)?